docs/releases.md

```diff
@@ -0,0 +1,60 @@
+# Releases
+
+Vite releases follow [Semantic Versioning](https://semver.org/). You can see the latest stable version of Vite in the [Vite npm package page](https://www.npmjs.com/package/vite).
+
+A full changelog of past releases is [available on GitHub](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md).
+
+::: tip note
+The next Vite Major release will happen after the Node 16 EOL in September.
+
+Check out the [Vite 5 discussion](https://github.com/vitejs/vite/discussions/12466) for more information.
+:::
+
+## Release Cycle​
+
+Vite does not have a fixed release cycle.
+
+- **Patch** releases are released as needed.
+- **Minor** releases always contain new features, with a typical time frame of ~2-3 months in between. Minor releases always go through a beta pre-release phase.
+- **Major** releases generally align with [Node.js EOL schedule](https://endoflife.date/nodejs), and will be announced ahead of time. These releases will go through an early discussion phase, and both alpha and beta pre-release phases.
+
+The latest minor in the previous Vite Major to the latest one will keep receiving important fixes and security patches. After that, it would only get updates if there are security concerns. We recommend updating Vite regularly. Check out the [Migration Guides](https://vitejs.dev/guide/migration.html) when you update to each Major.
+
+The Vite team partners with the main projects in the ecosystem to test new Vite versions before they are released through the [vite-ecosystem-ci project](https://github.com/vitejs/vite-ecosystem-ci). Most projects using Vite should be able to quickly offer support or migrate to new versions as soon as they are released.
+
+## Semantic Versioning Edge Cases
+
+### TypeScript Definitions​
+
+We may ship incompatible changes to TypeScript definitions between minor versions. This is because:
+- Sometimes TypeScript itself ships incompatible changes between minor versions, and we may have to adjust types to support newer versions of TypeScript.
+- Occasionally we may need to adopt features that are only available in a newer version of TypeScript, raising the minimum required version of TypeScript.
+- If you are using TypeScript, you can use a semver range that locks the current minor and manually upgrade when a new minor version of Vite is released.
+
+### esbuild
+
+[esbuild](https://esbuild.github.io/) is pre-1.0.0 and sometimes it has a breaking change we may need to include to have access to newer features and performance improvements. We may bump the esbuild's version in a Vite Minor.
+
+### Node.js non-LTS versions
+
+Non-LTS Node.js versions (odd-numbered) support is dropped automatically in the next Vite Minor to their [EOL](https://endoflife.date/nodejs).
```

Comment:
I'm not sure what this means by dropping in the next Vite minor, maybe we can clarify that we don't support odd numbered versions at all?
```suggestion
Non-LTS Node.js versions (odd-numbered) are not supported but may still work if the closest lower Node.js version is supported by Vite.
```

Reply:
I think Vite supports odd-numbered versions while they are active, no? It is true that we aren't testing in CI and your comment is accurate but I feel we shouldn't say they aren't supported.

Maybe something like this?

Non-LTS Node.js versions (odd-numbered) are not tested as part of Vite CI, but they should still work before their [EOL](https://endoflife.date/nodejs).

Reply:
Yeah that new wording sounds good to me. I'm not sure if we had supported odd-numbered versions in the past, breaking changes could happen often in them, and Vite anticipating the breaking changes feels risky as nodejs could easily also revert things in those versions' minors.